hail/python/hail/expr/expressions/typed_expressions.py

```diff
@@ -284,10 +285,10 @@ def group_by(self, f):
         Examples
         --------
 
-        >>> hl.eval(a.group_by(lambda x: x % 2 == 0))
+        >>> hl.eval(a.group_by(lambda x: x % 2 == 0))  # doctest: +NOTEST
```

Comment:
what's up with this one?

Reply:
the order of the elements in the dict below is not guaranteed :(

Reply:
it can change run to run I think